config/config.go

```diff
@@ -79,6 +79,8 @@ func init() {
 	cfg.SetDefault("analyzer.topology.probes", []string{})
 	cfg.SetDefault("analyzer.ssh_enabled", false)
 	cfg.SetDefault("analyzer.replication.debug", false)
+	cfg.SetDefault("analyzer.startup_capture", "")
+	cfg.SetDefault("analyzer.startup_capture_bpf", "")
```

Comment:
If these value by default are empty then these two lines can be omitted.

Reply:
hi,
yes, ok, I will remove them.
Thanks!

Reply:
I would rather do

```
  analyzer:
    startup:
       captures:
           <list of capture>
       alerts
           <list of alerts>

so that we can register more than one captures. I do not suggest to handle alerts now, just thinking